packages/config/lib/index.js

```diff
@@ -20,6 +20,7 @@ const breakingKeys = _.map(breakingOptions, 'name')
 const defaultValues = createIndex(options, 'name', 'defaultValue')
 const publicConfigKeys = _(options).reject({ isInternal: true }).map('name').value()
 const validationRules = createIndex(options, 'name', 'validation')
+const writeableOptions = options.filter((option) => option.isWriteable).map((option) => option.name).reduce((name, val) => ({ ...name, [val]: true }), {})
```

Comment:
Are you able to use the existing createIndex helper here?

Reply:
@emilyrohrbough I'm not sure createIndex would work here because it isn't trying to create an object with format [keyKey]: [valueKey]. I'm just trying to create a new object with format [option]: true for each writeable option